lib/main.dart

```diff
@@ -3,36 +3,37 @@ import 'package:flutter/material.dart';
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:get_it/get_it.dart';
+import 'package:vocel/common/repository/auth_inject.dart';
+import 'package:vocel/common/repository/auth_repository.dart';
 import 'package:vocel/simple_app.dart';
-// import 'amplifyconfiguration.dart';
+
+final getIt = GetIt.instance;
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
   // Might use later to override the translations based on a user's preferences
-  List? languages = await Devicelocale.preferredLanguages;
-  String? locale = await Devicelocale.currentLocale;
+  // List? languages = await Devicelocale.preferredLanguages;
```

Comment:
The first two lines here - should they be commented out, too?  It looks like you've got some code for dealing with the amplifyconfiguration.dart being missing and some for language.  Remove any commented code that isn't to be uncommented.